src/pages/Pools/FarmingPoolsMarquee.tsx

```diff
@@ -132,16 +132,6 @@ const FarmingPoolsMarquee = ({ tab }: { tab: string }) => {
 
 export default FarmingPoolsMarquee
 
-const IconWrapper = styled.div`
```

Comment:
Is this included? 🤔

Reply:
just remove unuse var.  not related main purpose of this PR :D